### ISSUE TITLE
Crash while reloading solution

### ISSUE THREAD

**Issue by MichaelGoulding** [Microsoft/PTVS#3045]
[Window Title]
devenv.exe

[Main Instruction]
An unexpected error occurred

[Content]
Please press Ctrl+C to copy the contents of this dialog and report this error to our issue tracker.

[^] Hide details  [Close]

[Expanded Information]
```
Build: 3.0.17255.1

System.InvalidOperationException: Failed to create entry for file
   at Microsoft.PythonTools.Intellisense.VsProjectAnalyzer.<AnalyzeFileAsync>d__82.MoveNext() in E:\A\_work\2\s\Python\Product\PythonTools\PythonTools\Intellisense\ProjectAnalyzer.cs:line 885
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.PythonTools.Intellisense.IntellisenseController.<ConnectSubjectBufferAsync>d__22.MoveNext() in E:\A\_work\2\s\Python\Product\PythonTools\PythonTools\Intellisense\IntellisenseController.cs:line 187
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Microsoft.PythonTools.Infrastructure.VSTaskExtensions.<HandleAllExceptions>d__5.MoveNext() in E:\A\_work\2\s\Python\Product\VsCommon\Infrastructure\VSTaskExtensions.cs:line 179
```

**Comment by zooba** [comment]
At a guess, this was a race condition with the (unnecessary) analyzer reset that we do. There's a TODO in our code to get more information about these failures, but probably the best response in ConnectSubjectBufferAsync is to get the analyzer (again) and try again.

**Comment by MichaelGoulding** [comment]
The happened after starting a rebase from the command line, seeing a conflict, then switching to Visual Studio who asked to reload the solution.  It then crashed.

**Comment by zooba** [comment]
Yep, that's consistent. Project reload is when we start running the code analyzer process, and right now that can include a mid-reload reset that would cause this.

**Comment by zooba** [comment]
This was fixed as part of #3050 (see IntellisenseController.cs changes)
